requirements.md

```diff
@@ -73,16 +90,22 @@ authorization it can be validated against relying-party stored instrument displa
 ## SPC Credentials
 
 * When more than one SPC Credential matches input data, the browser must choose the authenticator in the order of the input SPC Credential Identifiers.
+
+Q: Does it mean that if more than one SPC Cred IDs are valid on the given device/browser then the browser picks first on the list and does not present selection?
```

Comment:
I think this is an interesting question, but I'm not sure what giving a choice here to the user would do. It might be useful for us to think through where the multiple matching credentials may have come from (i.e. what are the journeys that led there), and what difference it makes to the user which one they select.

Reply:
I am going to remove this requirement and instead link to issue 69 for discussion.

Reply:
Resolving with that edit in place.